backend/baseclass.pm

```diff
@@ -1341,9 +1340,10 @@ sub stop_ssh_serial {
 
     my $ssh = $self->{serial};
     return undef unless $ssh;
-
+    bmwqemu::diag("Close SSH serial connection with " . $ssh->hostname);
```

Comment:
This is the "imperative" form and could confuse the reader to think that she would need to do anything. I guess you can state "Closing"? Also, as you are just concatenating the argument afterwards you can use single quotes.

Reply:
done